packages/grafana-ui/src/components/Select/SingleValue.tsx

```diff
@@ -75,14 +77,14 @@ export const SingleValue = (props: Props) => {
   );
 };
 
-const FadeWithImage = (props: { loading: boolean; imgUrl: string; styles: StylesType }) => {
+const FadeWithImage = (props: { loading: boolean; imgUrl: string; styles: StylesType; alt?: string }) => {
```

Comment:
Shouldn't `alt` be here required prop? It is not exported and only used here. And we know that we have `value` for it as it is required prop.

Reply:
good point!

Reply:
```suggestion
const FadeWithImage = (props: { loading: boolean; imgUrl: string; styles: StylesType; alt: string }) => {
```